CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#6964](https://github.com/rubocop-hq/rubocop/issues/6964): Set default `IgnoreCopDirectives` to `true` for `Metrics/LineLength`. ([@jdkaplan][])
```

Comment:
Ah, could you rebase with the latest master branch and move this entry to the `### Changes` section of `## master (unreleased)`?

Reply:
Whoops! I didn't even notice the sub-headings there. I'll do this now.